### ISSUE TITLE
Move the module to voxpupuli

### ISSUE THREAD

**Issue by vStone** [razorsedge/puppet-snmp#84]
Since you mentioned being swarmed with work, maybe we can move this module to voxpupuli so more eyes can keep it up to date (looking at puppet4 and strict_variables)

https://voxpupuli.org/docs/#migrating-a-module-to-voxpupuli

**Comment by dhollinger** [comment]
@razorsedge any thoughts? 

We would be more than willing to take it in and maintain it.

**Comment by elconas** [comment]
:+1:  do it :)

**Comment by blackknight36** [comment]
I guess I'll throw my +1 in here.  I'd love to see an active community maintaining a single module instead of 80 different forks.

**Comment by cedef** [comment]
Any news on this one ? If I understand correctly, voxpupuli is waiting on @razorsedge to follow the [migration guide](https://voxpupuli.org/docs/#migrating-a-module-to-voxpupuli).

**Comment by oranenj** [comment]
Can I somehow help make this happen? This module could use a bit of Puppet 4 updating, even though it does work well even as is.

**Comment by blackknight36** [comment]
Not sure what to do at this point.  I have a custom fork of this module which does provide puppet 4 support and I'd love to see the changes merged into this module.

**Comment by ardichoke** [comment]
Could we please make this happen? Alternatively, @blackknight36, could you take your fork and migrate it to the voxpupuli community?

**Comment by blackknight36** [comment]
@ardichoke I'll see what I can do.  I was waiting to see if any of my PRs would be merged in.  @razorsedge seems to have returned from the dead for a bit but it's been a few weeks without any response now.

**Comment by johanfleury** [comment]
Any updates?

**Comment by blackknight36** [comment]
I can talk to Vox Pupuli about moving this module over.  They normally like to work with the original module author but it seems like @razorsedge is unavailable for the most part.

**Comment by hdep** [comment]
Any news about moving this module over ?

**Comment by blackknight36** [comment]
@hdep I haven't had time to go through the process of moving to Voxpupuli but I do have my own fork which is still being maintained.  I've been considering publishing my fork on the puppet forge as well.

**Comment by blackknight36** [comment]
I've just published this module on the forge.  See https://forge.puppet.com/blackknight36/snmp for details.

**Comment by razorsedge** [comment]
Every time that I think I have the time and energy to work on this module, I end up not doing so.

I have begun the process of migrating to voxpupuli.  I will keep this issue informed as to its progress.

**Comment by blackknight36** [comment]
@razorsedge Thanks for your help and all of your work on this module.  I look forward to contributing more.

**Comment by johanfleury** [comment]
This is good news.

Thank you for all the work you’ve done on this module!

**Comment by razorsedge** [comment]
If you are reading this, the module ownership has changed.  I think a few more steps in the process have to be taken by Vox Pupuli staff.

**Comment by ubellavance** [comment]
Thanks for your work @razorsedge
